package-lock.json

```diff
@@ -21647,9 +21647,9 @@
             }
         },
         "typescript": {
-            "version": "4.4.3",
```

Comment:
This change needs to be reverted too. Otherwise it doesn't match the package.json.